setup.cfg

```diff
@@ -22,11 +22,35 @@ install_requires =
      typer
      ruamel.yaml
      jsonschema
+     jinja2
 
 [options.packages.find]
 # Don't include our tests directory in the distribution
 exclude = tests
 
+[options.extras_require]
+dev =
+    # Pinning black stops us having to allow pre-releases globally
+    black==21.9b0
+    # Flake version 4.0 and 4.0.1 do not like pytest: https://github.com/tholo/pytest-flake8/issues/81
+    flake8==3.9.2
+    # Pins to make lockfile usable on multiple Python versions and platforms
+    mypy
+    atomicwrites
+    typing-extensions
+    importlib-metadata
+    typed-ast
+    # Test and docs dependencies
+    pytest-cov
+    pytest-mypy
+    pytest-flake8
+    pytest-black
+    flake8-isort
+    isort>=5.0
+    sphinx-rtd-theme
+    sphinx-apischema
+    autopep8==1.5.7
```

Comment:
What's autopep8 used for?

Reply:
Not sure, guessing someone was previously using it as an alternative to black?

Reply:
I have removed it anyway...